pkg/management/postgres/metrics/collector.go

```diff
@@ -378,6 +387,66 @@ func (c QueryCollector) collect(conn *sql.DB, ch chan<- prometheus.Metric) error
 	return nil
 }
 
+// isCollectable checks if a query to collect metrics can be executed or not.
```

Comment:
Is there any reason why we are accepting multiple types?
In my opinion, we should only evaluate booleans, and it should be the responsibility of the query to return the intended type

Reply:
I agree with Armando. It's better to rely on booleans to do that.
True means it's collectible, false means it's not, and NULL means we don't know.
And if we don't know, perhaps it's better not to collect it.

Reply:
So to summarize we want to support ONLY bools so that even numbers are not tolerated. Also, do we want to force the developers to deliver a query that returns ONLY one column? If so we can simplify the whole logic with a single `QueryRow` and a `Scan` that accepts a bool variable.

Reply:
Yes, I think evaluating a bool should be enough

Reply:
Update the comment and the method accordingly. Now assume that the type is a `bool` if not we return `false` and log a warning.

Reply:
The updated code handles:
- `bool` checking the first row first column.
- no rows, returning `false`.
- null value, returning `false`.